modules/distribution/src/assembly/bin.xml

```diff
@@ -539,73 +420,82 @@
+                <include>**/entitlement.properties</include>
             </includes>
         </fileSet>
-
-        <!--creating an empty event sink directory-->
         <fileSet>
-            <directory>.</directory>
+            <directory>
+                ../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/conf/identity/identity-providers
+            </directory>
             <outputDirectory>
-                ${pom.artifactId}-${pom.version}/repository/deployment/server/eventsink
+                ${pom.artifactId}-${pom.version}/repository/conf/identity/identity-providers
             </outputDirectory>
-            <excludes>
-                <exclude>**/*</exclude>
-            </excludes>
+            <includes>
+                <include>**/default.xml</include>
+            </includes>
         </fileSet>
-
-        <!-- Start UES related artifacts -->
-
-
-        <!-- Dashboard portal Jaggery app. Should be copied to <DAS_HOME>/repository/deployment/server/jaggeryapps -->
-        <!--<fileSet>-->
-            <!--<directory>-->
-                <!--../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/deployment/server/jaggeryapps/portal-->
-            <!--</directory>-->
-            <!--<outputDirectory>${pom.artifactId}-${pom.version}/repository/deployment/server/jaggeryapps/portal</outputDirectory>-->
-            <!--<includes>-->
-                <!--<include>**/*</include>-->
-            <!--</includes>-->
-        <!--</fileSet>-->
-
-        <!-- END UES related dashboard artifacts -->
-
         <fileSet>
-            <directory>src/sso</directory>
+            <directory>
+                ../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/conf/identity/service-providers
+            </directory>
             <outputDirectory>
-                ${pom.artifactId}-${pom.version}/repository/deployment/server/jaggeryapps/sso
+                ${pom.artifactId}-${pom.version}/repository/conf/identity/service-providers
             </outputDirectory>
             <includes>
-                <include>**/*</include>
+                <include>**/default.xml</include>
+            </includes>
+        </fileSet>
+
+        <!-- Analytics related config -->
+        <fileSet>
+            <directory>
+                ../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/conf/analytics/
+            </directory>
+            <outputDirectory>${pom.artifactId}-${pom.version}/repository/conf/analytics</outputDirectory>
+            <includes>
+                <include>**/analytics-config.xml</include>
+                <include>**/rdbms-config.xml</include>
+                <!--<include>**/hbase-analytics-config.xml</include>-->
+                <include>**/analytics-data-config.xml</include>
+                <include>**/analytics-eventsink-config.xml</include>
             </includes>
         </fileSet>
     </fileSets>
 
+
+
```

Comment:
Already fixed in commit https://github.com/wso2-incubator/product-iot-server/commit/febdc36cba8b94c3ebcaba27947893ebb2865513